Xamarin.Forms.Controls/CoreGalleryPages/PickerCoreGalleryPage.cs

```diff
@@ -23,6 +23,17 @@ protected override void Build(StackLayout stackLayout)
 			var titleContainer = new ViewContainer<Picker>(Test.Picker.Title, new Picker());
 			titleContainer.View.Title = "Title";
 
+			var titleColorContainer = new ViewContainer<Picker>(Test.Picker.TitleColor, new Picker());
+			titleColorContainer.View.Title = "Title Color";
+			titleColorContainer.View.TitleColor = Color.Red;
+
+			var button = new Button() { Text = "Reset color to default" };
+			button.Clicked += (o, a) => {
+				titleColorContainer.View.ClearValue(Picker.TitleColorProperty);
+			};
+
+			titleColorContainer.ContainerLayout.Children.Add(button);
```

Comment:
Could you add another button to set the color (ex. Green). To verify that it is successfully set dynamically.

Reply:
Done